### PR TITLE
chore: add note about http(s) proxies

### PR DESCRIPTION
**Pull request by giacope**
Maybe it's not the best way to phrase it, but the idea is that only 1 proxy should be specified, either the HTTP or the HTTPS one. Open to suggestions 🙂

### CODE REVIEW

File: charts/cloudserver/values.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ proxy:
   # Additionally you can pass a CA certifcate that will be added to the trusted
   # certs. If the proxy URL is configured and caCert is true, helm
   # will look for a file named 'ca.crt' at the root path of this chart.
+  # Note: To avoid unexpected behavior, only specify an HTTP or an HTTPS proxy.
```
Comment (jonathan-gramain) [inline charts/cloudserver/values.yaml]: `only specify one of "http" or "https" proxy options`?

Comment (giacope) [reply charts/cloudserver/values.yaml]: Updated!

File: charts/cloudserver/values.yaml [hunk 2]
```diff
@@ -49,6 +49,7 @@ proxy:
   # Additionally you can pass a CA certifcate that will be added to the trusted
   # certs. If the proxy URL is configured and caCert is true, helm
   # will look for a file named 'ca.crt' at the root path of this chart.
+  # Note: To avoid unexpected behavior, only specify one of "http" or "https" proxy options.
```
Comment (jonathan-gramain) [inline charts/cloudserver/values.yaml]: minor but lets-do-this-while-its-hot: can you wrap the line so that it fits into 80 columns?

**Review by jonathan-gramain — APPROVED** [review]

**Review by philipyoo — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

File: charts/cloudserver/values.yaml (hunk 2 above)
Comment (giacope) [reply charts/cloudserver/values.yaml]: Done!

**Pull request merged**